index.html

```diff
@@ -26,6 +26,27 @@ <h3 class="title">Top Gun</h3>
         </button>
     </div>
 
+    <br />
```

Comment:
this line is deprecated please use different method for line space

Reply:
changed